.github/workflows/update-viablestrict.yml

```diff
@@ -0,0 +1,45 @@
+name: Update viable/strict
+
+on:
+  schedule:
+    - cron: 17,47 * * * *
```

Comment:
Not urgent to address in this PR, but good to have a comment explaining the cron